src/services/edge-application-error-responses-services/edit-error-responses-service.js

```diff
@@ -15,12 +15,22 @@ export const editErrorResponsesService = async (payload) => {
 }
 
 const adapt = (payload) => {
-  const errorResponses = payload.errorResponses.map((element) => {
-    return {
-      custom_status_code: element.customStatusCode?.toString() || null,
-      uri: element.uri || null,
-      timeout: element.timeout,
-      code: element.code
+  const errorResponses = []
```

Comment:
Suggestion: You could use the array filter instead:

```
const errorResponses = payload.errorResponses.map(element => ({
  custom_status_code: element.customStatusCode?.toString() || null,
  uri: element.uri || null,
  timeout: element.timeout,
  code: element.code
}));

const filteredResponses = errorResponses.filter(element => element.code === 'any');
const otherResponses = errorResponses.filter(element => element.code !== 'any');

const finalResponses = filteredResponses.concat(otherResponses);

```
But is only a suggestion, not a blocker.